build.sh

```diff
@@ -258,11 +258,10 @@ echo "Current directory: $(pwd)"
 echo "Build directory: ${build_dir}"
 echo "Install directory: ${install_dir}"
 
-if [ "$n_parallel"  -gt "10" ]; then
-    jobs=$(( (n_parallel / 10) * 9 ))
-else
-    jobs=16
-fi
+
+jobs=$(( (n_parallel -1)))
```

Comment:
Can remove the inner most () and keep a space before the ))

Reply:
I forgot the space